geometry_msgs/Vector3[] nodeCoordinates
bool[] occupiedStatus
uint16[] nodeDistances

